lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -137,7 +142,9 @@ export default function FilePickerApp({ onSubmit }) {
         <div className="FilePickerApp__left-col">Assignment content</div>
         <div className="FilePickerApp__right-col">
           {content ? (
-            <i data-testid="content-summary">{contentDescription(content)}</i>
+            <i data-testid="content-summary" style="word-break:break-all">
```

Comment:
Inline style here because I plan to convert this component over to utility (tailwind) styles shortly and don't want to create extra external CSS. Also, keeping this context closer to the developer is nice.

Reply:
(Updated: I rebased this branch onto `master` which now has tailwind support and updated this).